openpdf/src/main/java/com/lowagie/text/pdf/TrueTypeFont.java

```diff
@@ -51,10 +51,8 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
```

Comment:
Don't replace the explicit `java.util.CLASS` imports by a `java.util.*` import, that is not allowed by checkstyle options.